Albo1125.Common/CommonLibrary/ExtensionMethods.cs

```diff
@@ -446,6 +446,25 @@ public static void SetColors(this Vehicle v, VehicleColor color)
         {
             NativeFunction.Natives.SET_VEHICLE_COLOURS(v, (int)color.PrimaryColor, (int)color.SecondaryColor);
         }
+
+        /// <summary>
+        /// Randomise the license plate to avoid excessively frequent debug plates from showing.
+        /// </summary>
+        public static void RandomiseLicencePlate(Vehicle vehicle)
```

Comment:
Probably useful to make this an actual extension method by placing the `this` keyword in front of the first parameter:
`public static void RandomiseLicencePlate(this Vehicle vehicle)`

This integrates nicely with Visual Studio IntelliSense :)

Neat addition other than that!